### ISSUE TITLE
Private key and certificate get logged one buffer element per line in testing mode

### ISSUE THREAD

**Issue by Farbdose** [Daplie/letsencrypt-express#22]
When using 
```javascript
require('letsencrypt-express').testing()
```
opts.debug is set to true without any hint  for the user of doing so ... where and why ...
The log is unreadable because private key and certificate get printed one buffer element per line  

```javascript
[LEX] creating sniCallback {
  "debug": true,
  "configDir": "./letsencrypt",
  "webrootPath": "./letsencrypt/webroot",
  "httpsOptions": {
    "ciphers": "EECDH+AESGCM:EDH+AESGCM:AES256+EECDH:AES256+EDH",
    "cert": {
      "type": "Buffer",
      "data": [
        45,
        45,
        45,
         .
         .         
         .
         .
         .
         .
         .         
        45,
        45,
        10
      ]
    }
  },
  "letsencrypt": {
    "backend": {}
  },
  "privkeyPath": ":config/live/:hostname/privkey.pem",
  "fullchainPath": ":config/live/:hostname/fullchain.pem",
  "certPath": ":config/live/:hostname/cert.pem",
  "chainPath": ":config/live/:hostname/chain.pem",
  "server": "https://acme-staging.api.letsencrypt.org/directory"
}
```

**Comment by coolaj86** [comment]
`require('letsencrypt-express').testing()`

It's the `testing()` part that sets debug to true. Take away the testing and it will only show errors.

I suppose I could put a `.toJSON` on the cert object to make it not display. I agree that's a little annoying, but it's how node.js does it. I only recently added `httpsOptions` to the options object, which is necessary for use with raw http / https / http2 / spdy modules.

**Comment by Farbdose** [comment]
How about this instead d81a2d0919eb0870af98c62054e7d4067048fd0c ?

**Comment by Farbdose** [comment]
I would just print these long arrays in one line d81a2d0919eb0870af98c62054e7d4067048fd0c what do you think?

**Comment by coolaj86** [comment]
landed in v1.1.2

please test that it A) works and B) works to you liking.

:-)
